### PR TITLE
tests: queue tests fail

### LINKED ISSUE #115: Queue tests fail
OS: `Ubuntu 20.04`
Tarantool: `2.8.2-0-gfc96d10f5`

For now, all queue tests fail.

```
$ cd queue/
$
$ mkdir snap xlog
$
$ tarantoolctl rocks install queue 1.1.0
$
$ tarantool ./config.lua > tarantool.log 2>&1 &
[1] 988812
$
$ cat tarantool.log
Deprecated option snap_dir, please use memtx_dir instead
2021-12-02 08:51:16.580 [988046] main/103/config.lua C> Tarantool 2.8.2-0-gfc96d10f5
2021-12-02 08:51:16.580 [988046] main/103/config.lua C> log level 5
2021-12-02 08:51:16.580 [988046] main/103/config.lua I> wal/engine cleanup is paused
2021-12-02 08:51:16.580 [988046] main/103/config.lua I> mapping 268435456 bytes for memtx tuple arena...
2021-12-02 08:51:16.580 [988046] main/103/config.lua I> Actual slab_alloc_factor calculated on the basis of desired slab_alloc_factor = 1.044274
2021-12-02 08:51:16.580 [988046] main/103/config.lua I> mapping 134217728 bytes for vinyl tuple arena...
2021-12-02 08:51:16.582 [988046] main/103/config.lua I> instance uuid a227e131-f9eb-4e7f-8b80-a4536bca71f7
2021-12-02 08:51:16.582 [988046] main/103/config.lua I> tx_binary: stopped
2021-12-02 08:51:16.582 [988046] main/103/config.lua I> tx_binary: bound to 0.0.0.0:3013
2021-12-02 08:51:16.582 [988046] main/103/config.lua I> initializing an empty data directory
2021-12-02 08:51:16.592 [988046] main/103/config.lua I> assigned id 1 to replica a227e131-f9eb-4e7f-8b80-a4536bca71f7
2021-12-02 08:51:16.592 [988046] main/103/config.lua I> cluster uuid c32cf3a6-70b5-485c-b9e5-913ad7300151
2021-12-02 08:51:16.592 [988046] snapshot/101/main I> saving snapshot `snap/00000000000000000000.snap.inprogress'
2021-12-02 08:51:16.607 [988046] snapshot/101/main I> done
2021-12-02 08:51:16.607 [988046] main/103/config.lua I> ready to accept requests
2021-12-02 08:51:16.607 [988046] main/104/gc I> wal/engine cleanup is resumed
2021-12-02 08:51:16.607 [988046] main/103/config.lua I> set 'log_level' configuration option to 5
2021-12-02 08:51:16.607 [988046] main/105/checkpoint_daemon I> scheduled next checkpoint for Thu Dec  2 10:12:11 2021
2021-12-02 08:51:16.608 [988046] main/103/config.lua I> set 'listen' configuration option to "3013"
2021-12-02 08:51:16.608 [988046] main/103/config.lua I> set 'log_format' configuration option to "plain"
2021-12-02 08:51:16.618 [988046] main schema.lua:2498 E> ER_NO_SUCH_SPACE: Space '_queue_taken' does not exist
2021-12-02 08:51:16.618 [988046] main F> fatal error, exiting the event loop
```

According to [this](https://github.com/tarantool/queue/tree/7d05b6db5b0d596db4281d53b57811e156234a0d#fields-of-the-_queue_consumers-space), `_queue_taken` is deprecated. So let's try to use `_queue_taken_2`. Open the `config.lua` file and change `box.schema.user.grant('test', 'read,write', 'space', '_queue_taken')` to `box.schema.user.grant('test', 'read,write', 'space', '_queue_taken_2')`.

```
$ ps aux | awk '/tarantool/{print $2}' | head -1 | xargs kill
$
$ rm -rf snap xlog && mkdir snap xlog
$
$ tarantool ./config.lua > tarantool.log 2>&1 &
[1] 988893
$
$ cat tarantool.log
Deprecated option snap_dir, please use memtx_dir instead
2021-12-02 09:10:55.567 [988893] main/103/config.lua C> Tarantool 2.8.2-0-gfc96d10f5
2021-12-02 09:10:55.567 [988893] main/103/config.lua C> log level 5
2021-12-02 09:10:55.567 [988893] main/103/config.lua I> wal/engine cleanup is paused
2021-12-02 09:10:55.567 [988893] main/103/config.lua I> mapping 268435456 bytes for memtx tuple arena...
2021-12-02 09:10:55.567 [988893] main/103/config.lua I> Actual slab_alloc_factor calculated on the basis of desired slab_alloc_factor = 1.044274
2021-12-02 09:10:55.567 [988893] main/103/config.lua I> mapping 134217728 bytes for vinyl tuple arena...
2021-12-02 09:10:55.568 [988893] main/103/config.lua I> instance uuid 8fead1ef-bff9-4e30-a857-c47ce4833944
2021-12-02 09:10:55.568 [988893] main/103/config.lua I> tx_binary: stopped
2021-12-02 09:10:55.569 [988893] main/103/config.lua I> tx_binary: bound to 0.0.0.0:3013
2021-12-02 09:10:55.569 [988893] main/103/config.lua I> initializing an empty data directory
2021-12-02 09:10:55.579 [988893] main/103/config.lua I> assigned id 1 to replica 8fead1ef-bff9-4e30-a857-c47ce4833944
2021-12-02 09:10:55.579 [988893] main/103/config.lua I> cluster uuid 1f211e77-88c4-4a38-9875-b4105ee1f45c
2021-12-02 09:10:55.580 [988893] snapshot/101/main I> saving snapshot `snap/00000000000000000000.snap.inprogress'
2021-12-02 09:10:55.589 [988893] snapshot/101/main I> done
2021-12-02 09:10:55.590 [988893] main/103/config.lua I> ready to accept requests
2021-12-02 09:10:55.590 [988893] main/104/gc I> wal/engine cleanup is resumed
2021-12-02 09:10:55.590 [988893] main/103/config.lua I> set 'log_level' configuration option to 5
2021-12-02 09:10:55.590 [988893] main/105/checkpoint_daemon I> scheduled next checkpoint for Thu Dec  2 10:52:58 2021
2021-12-02 09:10:55.590 [988893] main/103/config.lua I> set 'listen' configuration option to "3013"
2021-12-02 09:10:55.590 [988893] main/103/config.lua I> set 'log_format' configuration option to "plain"
2021-12-02 09:10:55.599 [988893] main C> entering the event loop
$
$ go clean -testcache && go test -v
=== RUN   TestFifoQueue
--- FAIL: TestFifoQueue (0.00s)
    queue_test.go:40: Failed to create queue: Create access to space 'test_queue' is denied for user 'test' (0x2a)
=== RUN   TestFifoQueue_GetExist_Statistic
--- FAIL: TestFifoQueue_GetExist_Statistic (0.00s)
    queue_test.go:65: Failed to create queue: Create access to space 'test_queue' is denied for user 'test' (0x2a)
=== RUN   TestFifoQueue_Put
--- FAIL: TestFifoQueue_Put (0.00s)
    queue_test.go:116: Failed to create queue: Create access to space 'test_queue' is denied for user 'test' (0x2a)
=== RUN   TestFifoQueue_Take
--- FAIL: TestFifoQueue_Take (0.03s)
    queue_test.go:159: Failed to create queue: Create access to space 'test_queue' is denied for user 'test' (0x2a)
=== RUN   TestFifoQueue_TakeTyped
--- FAIL: TestFifoQueue_TakeTyped (0.00s)
    queue_test.go:255: Failed to create queue: Create access to space 'test_queue' is denied for user 'test' (0x2a)
=== RUN   TestFifoQueue_Peek
--- FAIL: TestFifoQueue_Peek (0.00s)
    queue_test.go:332: Failed to create queue: Create access to space 'test_queue' is denied for user 'test' (0x2a)
=== RUN   TestFifoQueue_Bury_Kick
--- FAIL: TestFifoQueue_Bury_Kick (0.00s)
    queue_test.go:391: Failed to create queue: Create access to space 'test_queue' is denied for user 'test' (0x2a)
=== RUN   TestFifoQueue_Delete
--- FAIL: TestFifoQueue_Delete (0.00s)
    queue_test.go:478: Failed to create queue: Create access to space 'test_queue' is denied for user 'test' (0x2a)
=== RUN   TestFifoQueue_Release
--- FAIL: TestFifoQueue_Release (0.00s)
    queue_test.go:553: Failed to create queue: Create access to space 'test_queue' is denied for user 'test' (0x2a)
=== RUN   TestTtlQueue
--- FAIL: TestTtlQueue (0.00s)
    queue_test.go:643: Failed to create queue: Create access to space 'test_queue' is denied for user 'test' (0x2a)
=== RUN   TestTtlQueue_Put
--- FAIL: TestTtlQueue_Put (0.00s)
    queue_test.go:700: Failed to create queue: Create access to space 'test_queue' is denied for user 'test' (0x2a)
=== RUN   TestUtube_Put
--- FAIL: TestUtube_Put (0.00s)
    queue_test.go:772: Failed to create queue: Create access to space 'test_utube' is denied for user 'test' (0x2a)
FAIL
exit status 1
FAIL	github.com/tarantool/go-tarantool/queue	0.063s
```

### PR DESCRIPTION
**Pull request by vr009**
tests: queue tests fix 

Before this patch, queue tests failed due to impossibility to use _queue_taken,
which was deprecated in tarantool/queue [1]. Switch to _queue_taken_2 fixed it. 

Create access and some other actions were denied for test user. 
Additional grants fixed this problem.

The test `TestUtube_Put` was affected by replacing timeout in the call 
of `TakeTimeout()` with timeout of connection structure. For clarification 
of this effect see annotation for function `TakeTimeout()` [2]. That was 
fixed by making connection timeout longer than the timeout in the function call.

The step in testing.yml was uncommented for enabling queue tests in CI.

1. [queue/_queue_taken_2](https://github.com/tarantool/queue/tree/7d05b6db5b0d596db4281d53b57811e156234a0d#fields-of-the-_queue_consumers-space)
2. https://github.com/tarantool/go-tarantool/blob/master/queue/queue.go#L35

Fixes #115

### CODE REVIEW

File: .github/workflows/reusable_testing.yml [hunk 1]
```diff
@@ -45,15 +45,15 @@ jobs:
...
-#          kill $TNT_PID
+      - name: Run queue tests
+        working-directory: ./queue
+        run: |
+          mkdir snap xlog
+          tarantoolctl rocks install queue 1.1.0
+          TNT_PID=$(tarantool ./config.lua > tarantool.log 2>&1 & echo $!)
+          go clean -testcache && go test -v
+          kill $TNT_PID
+        if: ${{ !startsWith(env.TNT_VERSION, 'Tarantool 1.10') }}
```
Comment (DifferentialOrange) [inline .github/workflows/reusable_testing.yml:56]: What's the deal with 1.10?

Comment (vr009) [reply .github/workflows/reusable_testing.yml:56]: I've clashed with some problem on calling `box.cfg`  in tarantool 1.10. That looks like some problem with initializing
collations. See the logs:
```
2022-01-26 10:13:45.506 [1706] main/101/config.lua C> Tarantool 1.10.12-22-g73a90f94c
2022-01-26 10:13:45.507 [1706] main/101/config.lua C> log level 5
2022-01-26 10:13:45.507 [1706] main/101/config.lua I> mapping 268435456 bytes for memtx tuple arena...
2022-01-26 10:13:45.507 [1706] main/101/config.lua I> Actual slab_alloc_factor calculated on the basis of desired slab_alloc_factor = 1.044274
2022-01-26 10:13:45.507 [1706] main/101/config.lua I> mapping 134217728 bytes for vinyl tuple arena...
2022-01-26 10:13:45.508 [1706] main/101/config.lua I> instance uuid a5fec003-cf8a-45ed-9ff5-3931e118b2cb
2022-01-26 10:13:45.508 [1706] iproto/101/main I> binary: bound to [::]:3013
2022-01-26 10:13:45.508 [1706] main/101/config.lua I> initializing an empty data directory
2022-01-26 10:13:45.509 [1706] main/101/config.lua alter.cc:2380 E> ER_CANT_CREATE_COLLATION: Failed to initialize collation: unknown collation type.
2022-01-26 10:13:45.509 [1706] main/101/config.lua F> can't initialize storage: Failed to initialize collation: unknown collation type.
```
I've noticed that in some steps there is the same problem and these steps constraint test running on tarantool 1.10. So I've done the same until we discuss it and maybe fix this problem.

**Comment by DifferentialOrange** [comment]
Btw, commit name is confusing: queue tests are fixed now, they fail no more.

**Comment by vr009** [comment]
> Btw, commit name is confusing: queue tests are fixed now, they fail no more.

Fixed.

File: .github/workflows/reusable_testing.yml (hunk 1 above)
Comment (DifferentialOrange) [reply .github/workflows/reusable_testing.yml:56]: This is weird. How collations related to queue test? I don't found any explicit collation specifications both here and in tarantool/queue.

Comment (DifferentialOrange) [reply .github/workflows/reusable_testing.yml:56]: Maybe you've tried to start Tarantool 1.10 over Tarantool 2.x snapshots?

Comment (vr009) [reply .github/workflows/reusable_testing.yml:56]: Idk, I thought the same, but I've cleared the directory before start. The effect is the same.

Comment (DifferentialOrange) [reply .github/workflows/reusable_testing.yml:56]: Well, seems like this was really the issue.
Snapshots from 2.x:
```
tarantool ./config.lua
Deprecated option snap_dir, please use memtx_dir instead
2022-01-26 12:58:04.606 [23294] main/101/config.lua C> Tarantool Enterprise 1.10.11-0-gf0b0e7ecf-r423
2022-01-26 12:58:04.607 [23294] main/101/config.lua C> log level 5
2022-01-26 12:58:04.607 [23294] main/101/config.lua I> mapping 268435456 bytes for memtx tuple arena...
2022-01-26 12:58:04.607 [23294] main/101/config.lua I> Actual slab_alloc_factor calculated on the basis of desired slab_alloc_factor = 1.044274
2022-01-26 12:58:04.608 [23294] main/101/config.lua I> mapping 134217728 bytes for vinyl tuple arena...
2022-01-26 12:58:04.610 [23294] main/101/config.lua I> instance uuid 2599ce5f-b33c-44eb-acf2-a8cadbf8f3d4
2022-01-26 12:58:04.610 [23294] main/101/config.lua I> instance vclock {1: 49}
2022-01-26 12:58:04.611 [23294] iproto/101/main I> binary: bound to 0.0.0.0:3013
2022-01-26 12:58:04.611 [23294] main/101/config.lua I> recovery start
2022-01-26 12:58:04.611 [23294] main/101/config.lua I> recovering from `snap/00000000000000000000.snap'
2022-01-26 12:58:04.612 [23294] main/101/config.lua I> cluster uuid 406c1498-783a-495d-b8a3-8601ad42240b
2022-01-26 12:58:04.612 [23294] main/101/config.lua alter.cc:2380 E> ER_CANT_CREATE_COLLATION: Failed to initialize collation: unknown collation type.
2022-01-26 12:58:04.612 [23294] main/101/config.lua F> can't initialize storage: Failed to initialize collation: unknown collation type.
```

Clean up dirs
```
rm -rf ./snap
rm -rf ./xlog
mkdir snap
mkdir xlog
```

```
tarantool ./config.lua 
Deprecated option snap_dir, please use memtx_dir instead
2022-01-26 12:58:27.969 [23309] main/101/config.lua C> Tarantool Enterprise 1.10.11-0-gf0b0e7ecf-r423
2022-01-26 12:58:27.969 [23309] main/101/config.lua C> log level 5
2022-01-26 12:58:27.969 [23309] main/101/config.lua I> mapping 268435456 bytes for memtx tuple arena...
2022-01-26 12:58:27.970 [23309] main/101/config.lua I> Actual slab_alloc_factor calculated on the basis of desired slab_alloc_factor = 1.044274
2022-01-26 12:58:27.970 [23309] main/101/config.lua I> mapping 134217728 bytes for vinyl tuple arena...
2022-01-26 12:58:27.971 [23309] main/101/config.lua I> instance uuid d82f1274-2982-4c55-9fec-bba08fdd030a
2022-01-26 12:58:27.971 [23309] iproto/101/main I> binary: bound to 0.0.0.0:3013
2022-01-26 12:58:27.971 [23309] main/101/config.lua I> initializing an empty data directory
2022-01-26 12:58:27.974 [23309] main/101/config.lua I> assigned id 1 to replica d82f1274-2982-4c55-9fec-bba08fdd030a
2022-01-26 12:58:27.974 [23309] main/101/config.lua I> cluster uuid 0d32e3f4-0f83-421c-b44a-a8715b0ba93f
2022-01-26 12:58:27.974 [23309] snapshot/101/main I> saving snapshot `snap/00000000000000000000.snap.inprogress'
2022-01-26 12:58:27.976 [23309] snapshot/101/main I> done
2022-01-26 12:58:27.977 [23309] main/101/config.lua I> ready to accept requests
2022-01-26 12:58:27.977 [23309] main/107/checkpoint_daemon I> started
2022-01-26 12:58:27.977 [23309] main/107/checkpoint_daemon I> scheduled the next snapshot at Wed Jan 26 14:56:39 2022
2022-01-26 12:58:27.987 [23309] main schema.lua:1817 E> ER_NO_SUCH_SPACE: Space '_trigger' does not exist
2022-01-26 12:58:27.987 [23309] main F> fatal error, exiting the event loop
```
Commented row with `_trigger`
```
tarantool ./config.lua 
Deprecated option snap_dir, please use memtx_dir instead
2022-01-26 12:59:15.458 [23351] main/101/config.lua C> Tarantool Enterprise 1.10.11-0-gf0b0e7ecf-r423
2022-01-26 12:59:15.458 [23351] main/101/config.lua C> log level 5
2022-01-26 12:59:15.459 [23351] main/101/config.lua I> mapping 268435456 bytes for memtx tuple arena...
2022-01-26 12:59:15.459 [23351] main/101/config.lua I> Actual slab_alloc_factor calculated on the basis of desired slab_alloc_factor = 1.044274
2022-01-26 12:59:15.459 [23351] main/101/config.lua I> mapping 134217728 bytes for vinyl tuple arena...
2022-01-26 12:59:15.459 [23351] main/101/config.lua I> instance uuid d82f1274-2982-4c55-9fec-bba08fdd030a
2022-01-26 12:59:15.459 [23351] main/101/config.lua I> instance vclock {1: 36}
2022-01-26 12:59:15.459 [23351] iproto/101/main I> binary: bound to 0.0.0.0:3013
2022-01-26 12:59:15.460 [23351] main/101/config.lua I> recovery start
2022-01-26 12:59:15.460 [23351] main/101/config.lua I> recovering from `snap/00000000000000000000.snap'
2022-01-26 12:59:15.460 [23351] main/101/config.lua I> cluster uuid 0d32e3f4-0f83-421c-b44a-a8715b0ba93f
2022-01-26 12:59:15.461 [23351] main/101/config.lua I> assigned id 1 to replica d82f1274-2982-4c55-9fec-bba08fdd030a
2022-01-26 12:59:15.461 [23351] main/101/config.lua I> recover from `xlog/00000000000000000000.xlog'
2022-01-26 12:59:15.461 [23351] main/101/config.lua I> done `xlog/00000000000000000000.xlog'
2022-01-26 12:59:15.461 [23351] main/101/config.lua I> recover from `xlog/00000000000000000036.xlog'
2022-01-26 12:59:15.461 [23351] main/101/config.lua I> done `xlog/00000000000000000036.xlog'
2022-01-26 12:59:15.461 [23351] main/101/config.lua I> ready to accept requests
2022-01-26 12:59:15.461 [23351] main/101/config.lua C> leaving orphan mode
2022-01-26 12:59:15.461 [23351] main/107/checkpoint_daemon I> started
2022-01-26 12:59:15.462 [23351] main/107/checkpoint_daemon I> scheduled the next snapshot at Wed Jan 26 14:02:39 2022
2022-01-26 12:59:15.462 [23351] main C> entering the event loop
```

So the only error I've got for Tarantool 1.10 started from clean work dirs is unknown `_trigger` space.

Comment (DifferentialOrange) [reply .github/workflows/reusable_testing.yml:56]: I see you removed `Tarantool != 1.10` constraint in "testing" workflow, but you're forgot to remove it in "reusable_testing"

Comment (DifferentialOrange) [reply .github/workflows/reusable_testing.yml:56]: Well, your approach to add `_trigger` space if it doesn't exists works, but I don't think it's correct. `_trigger` is a space added in 2.x for internal needs of SQL: https://github.com/tarantool/tarantool/commit/7cb3d2cfe6d2478dcf4a87527eb6e580ade90d6b , so it is unsurprising that we don't have `_trigger` in 1.10. But AFAIK we don't use sql in queue and this grants are useless for us (and for tests), so we can just remove `box.schema.user.grant('test','read','space','_trigger')`

Comment (vr009) [reply .github/workflows/reusable_testing.yml:56]: That sounds reasonable, but if I'm removing this grant I get the next output in tests:
```sh
=== RUN   TestFifoQueue
    queue_test.go:46: Failed drop queue: Read access to space '_trigger' is denied for user 'test' (0x2a)
--- FAIL: TestFifoQueue (0.00s)
=== RUN   TestFifoQueue_GetExist_Statistic
    queue_test.go:72: Failed drop queue: Read access to space '_trigger' is denied for user 'test' (0x2a)
--- FAIL: TestFifoQueue_GetExist_Statistic (0.00s)
=== RUN   TestFifoQueue_Put
    queue_test.go:124: Failed drop queue: Read access to space '_trigger' is denied for user 'test' (0x2a)
--- FAIL: TestFifoQueue_Put (0.00s)
=== RUN   TestFifoQueue_Take
    queue_test.go:167: Failed drop queue: Read access to space '_trigger' is denied for user 'test' (0x2a)
--- FAIL: TestFifoQueue_Take (0.00s)
=== RUN   TestFifoQueue_TakeTyped
    queue_test.go:299: Task data after take not equal with example. &queue_test.customData{customField:""} != &queue_test.customData{customField:"put_data"}
    queue_test.go:302: Task data after take not equal with example. &queue_test.customData{customField:""} != &queue_test.customData{customField:"put_data"}
    queue_test.go:263: Failed drop queue: Read access to space '_trigger' is denied for user 'test' (0x2a)

```

**Comment by DifferentialOrange** [comment]
Sorry, I removed the branch and closed the issue by mistake. Everything should be restored now.

File: .github/workflows/reusable_testing.yml (hunk 1 above)
Comment (DifferentialOrange) [reply .github/workflows/reusable_testing.yml:56]: Sorry, it seems that you're correct and my verification with removing `_trigger` grants were wrong. Seems like I ran tests without cleaning cache.

Nonetheless, I think creating `_trigger` if it not exists is a bad pattern. Please, use
```lua
if box.space._trigger ~= nil then
    box.schema.user.grant('test', 'read', 'space', '_trigger')
end
```
for `_trigger` and use the same for all tarantool internal spaces (`_fk_constraint`, `_ck_constraint`, `_func_index`).

Comment (vr009) [reply .github/workflows/reusable_testing.yml]: You are right, that looks better. Fixed.

Comment (vr009) [reply .github/workflows/reusable_testing.yml]: The constraint `Tarantool != 1.10` is also removed in reusable_testing.

**Comment by ligurio** [comment]
For clarity: in module queue space `queue_taken` has been renamed to `queue_taken_2` in commit https://github.com/tarantool/queue/commit/271bc971f0d48204ce5b98b5fa139b8fd33db748, issue https://github.com/tarantool/queue/issues/85.

**Review by ligurio — CHANGES_REQUESTED** [review]

**Review by DifferentialOrange — APPROVED** [review]
I have no objections

File: queue/config.lua [hunk 2]
```diff
@@ -1,4 +1,4 @@
-queue = require 'queue'
+queue = require('queue')
```
[the remark this answers is not in the document]
Comment (vr009) [reply queue/config.lua]: Rolled back for now. Are you sure that those changes must be related to this PR? If it is okay for all, I add them. But I could do it in a new PR.

File: queue/queue_test.go [hunk 3]
```diff
@@ -691,9 +692,10 @@ func TestTtlQueue_Put(t *testing.T) {
...
 	name := "test_queue"
 	cfg := queue.Cfg{
-		Temporary: true,
-		Kind:      queue.FIFO_TTL,
-		Opts:      queue.Opts{Ttl: 5 * time.Second},
+		Temporary:   true,
+		Kind:        queue.FIFO_TTL,
+		Opts:        queue.Opts{Ttl: 5 * time.Second},
+		IfNotExists: true,
 	}
```
[the remark this answers is not in the document]
Comment (vr009) [reply queue/queue_test.go]: I removed this option at all, cause there is no need to have it there. It seems that during fixing tests I had some situation when the `test_queue` was not deleted. That leaded to a failure of trying to create an existing queue. Now all tests work well without this option.

File: queue/queue_test.go [hunk 4]
```diff
@@ -13,7 +13,7 @@ import (
 
 var server = "127.0.0.1:3013"
 var opts = Opts{
-	Timeout: 500 * time.Millisecond,
+	Timeout: 2500 * time.Millisecond,
```
[the remark this answers is not in the document]
Comment (vr009) [reply queue/queue_test.go:16]: In the last test there is a situation, when the timeout of connection shadows the timout of a query which is waiting for blocked queue. This is notated [here](https://github.com/tarantool/go-tarantool/blob/master/queue/queue.go#L35). This behavior breaks the test.

File: queue/config.lua (hunk 2 above)
[the remark this answers is not in the document]
Comment (ligurio) [reply queue/config.lua]: I would choose a variant with a separate PR to don't block this one.

**Review by ligurio — APPROVED** [review]
LGTM

**Pull request merged**